components/dashboard/src/projects/Project.tsx

```diff
@@ -34,7 +36,7 @@ export default function () {
 
     useEffect(() => {
         updateProject();
-    }, [ teams, team ]);
```

Comment:
@jankeromnes, do you know the reason, why this should be triggered on both events? The question is also if there is a chance that this is run twice.

Reply:
> @jankeromnes, do you know the reason, why this should be triggered on both events?

- If `team` changes, `await getGitpodService().server.getTeamProjects(team.id)` below will return something else
- If `teams` change, maybe a team got deleted or so, so we should probably update again?

> The question is also if there is a chance that this is run twice.

I'm not sure, why?

Reply:
I think it's just sufficient to depend on `teams`, because `team` is derived directly from `location` and `teams` state. Also, `team` isn't a React state, is it? It might be just ignored. 🤷🏻‍♂️  Having said that, I've no longer any concerns about double execution. ✔️

Reply:
Is it possible that `teams` doesn't change, but the currently selected `team` changes? (E.g. based on the URL, or if the user opens a non-team project?)